src/pybind/mgr/dashboard_v2/controllers/auth.py

```diff
@@ -65,7 +70,7 @@ def password_hash(password, salt_password=None):
     def check_auth():
         username = cherrypy.session.get(Auth.SESSION_KEY)
         if not username:
-            Auth.logger.debug('Unauthorized access to {}'.format(cherrypy.url(
+            logger.debug('Unauthorized access to {}'.format(cherrypy.url(
```

Comment:
Don't use `format` string when logging, you can call the logger as:
```
logger.debug('Unauthorized access to %s', cherrypy.url(relative='server'))
```

Reply:
What is the benefit of your change?